routes/stocks.js

```diff
@@ -62,6 +62,6 @@ router.get('/', fetchStocks)
  *           schema:
  *             $ref: '#/components/schemas/errors/badImplementation'
  */
-router.post('/', authenticate, authorization, createStock, addNewStock)
+router.post('/', authenticate, authorizeUser('superUser'), createStock, addNewStock)
```

Comment:
Can we have the strings `'superUser', 'appOwner', etc` defined as a constant and then use it here. We might mess up with the strings sometimes leading to an issue!